docs/roles/secretary.rst

```diff
@@ -9,22 +9,33 @@ This page explains more about the responsibilities and role of the secretary.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The secretary is responsible for scheduling, advertising, and recording RITlug general and eboard meetings.
```

Comment:
I think we should expand this section. Based off the responsibilities below, I suggest this rewrite:

---

The secretary is responsible for record maintenance and organizational upkeep of RITlug. Their primary tool as an eboard member is the [Runbook](https://github.com/RITlug/runbook). Part of their role is to monitor the Runbook and ensure current practices by RITlug eboard members are accurately documented in the Runbook. They do not exclusively write documentation, but work with other eboard members to identify areas of need. Additionally, the secretary also builds the agenda for eboard meetings. This role can be considered the "_librarian_" or "_information architect_" of RITlug.

Reply:
I am plus one to this update.

Reply:
Thanks, this is definitely helpful and more focused to this role.